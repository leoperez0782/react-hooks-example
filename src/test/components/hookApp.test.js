import React from 'react';
import { shallow} from "enzyme";

import { HookApp } from '../../HookApp';

describe('Probando <HookApp />', ()=>{

    test('Debe renderizar correctamente', ()=>{
        const wrapper = shallow(<HookApp />);

        expect( wrapper ).toMatchSnapshot();
    })
})