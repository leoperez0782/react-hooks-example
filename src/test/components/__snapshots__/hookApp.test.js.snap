// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando <HookApp /> Debe renderizar correctamente 1`] = `
<div>
  <h1>
    Hola Mundo!!!!
  </h1>
</div>
`;
